src/App.tsx

```diff
@@ -15,9 +15,23 @@ import { SingleJob } from "./pages/SingleJob/SingleJob";
 const App: React.FC<{}> = (): React.ReactElement => {
 	const [loggedIn, setLoggedIn] = React.useState(false);
```

Comment:
could we not just import { useState } from "react" to avoid having to declare React.something across multiple lines?

Reply:
It's mainly there to be explicit, that's my preference if I'm honest

Reply:
fair, I can't argue against that